### PR TITLE
Add wait flag for extractions.

### PR DESCRIPTION
**Pull request by kbirk**

### CODE REVIEW

**Review by dgauldie — APPROVED** [review]

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java [hunk 1]
```diff
@@ -797,14 +798,23 @@ public ResponseEntity<Void> variableExtractions(
 	)
 	public ResponseEntity<Void> pdfExtractions(
 		@RequestParam("document-id") final UUID documentId,
-		@RequestParam(name = "domain", defaultValue = "epi") final String domain,
-		@RequestParam(name = "project-id", required = false) final UUID projectId
+		@RequestParam(name = "project-id", required = false) final UUID projectId,
+		@RequestParam(name = "wait", required = false, defaultValue = "false") final Boolean wait
```
Comment (YohannParis) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java:802]: Could we use the same ASYNC/SYNC nomenclature used for taskrunner?

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java (hunk 1 above)
Comment (dvince2) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java:802]: seconded!

File: packages/server/src/main/resources/messages.properties [hunk 2]
```diff
@@ -29,6 +29,7 @@ document.not-found = We couldn't find the requested document. Please verify that
 document.unable-to-delete = We couldn't delete the document. Please try again later.
 document.unable-to-update = We couldn't update the document. Please try again later.
 document.text-length-exceeded = This document exceeds our 600,000-word limit. If you still need to analyze this document, contact support.
+document.extracton.failed = "Document extraction failed. Please try again later."
```
Comment (dvince2) [inline packages/server/src/main/resources/messages.properties:32]: You don't need the ""'s here - i think they'll show up in the message.

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java (hunk 1 above)
Comment (kbirk) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java:802]: Sure

**Pull request merged**